### ISSUE TITLE
Check out this cool lightning node setup guide

### ISSUE THREAD

**Issue by thecurtic** [AZ-Bitcoin-Network/phoenix-bitdevs-events#7]
why this is cool and people should know about it:

This is a walk-through built by Leo, a Vancouver based lightning developer. It goes over how to setup a lightning node on your own hardware or a VPS.

www.nodeacademy.org

**Comment by bijeebuss** [comment]
This looks awesome, I definitely want to do a workshop in the future about how to setup a node

**Comment by sntslol** [comment]
Yeah this is niceeeeee. Let's cover this one exclusively on one of our bitdevs. Maybe @bijeebuss and I can take turns after LN 101 at the next BitDevs?
